docs/book/src/clusterctl/developers.md

```diff
@@ -178,8 +176,8 @@ When selecting the `--kubernetes-version`, ensure that the `kindest/node`
 image is available.
 
 For example, on [docker hub][kind-docker-hub] there is no
-image for version `v1.21.2`, therefore creating a CAPD workload cluster with
-`--kubernetes-version=v1.21.2` will fail. See [issue 3795] for more details.
+image for version `v1.21.3`, therefore creating a CAPD workload cluster with
```

Comment:
Not really important, but v1.21.2 exists now. I suppose when a new v1.21 version is published it will be at least v1.21.5

Reply:
Let's make the entire sentence more generic. e.g.
For examples, assuming that in in docker hub there is no an image for version vX.Y.Z, then creating a ... will fail

Reply:
Done